src/main/java/com/revo/deployr/client/util/RDataUtil.java

```diff
@@ -242,6 +252,8 @@ private static RData buildVectorFromJSON(String name, Map json) {
                             ex.printStackTrace();
                             dates.add(null);
                         }
+                    }else{
```

Comment:
we have two places, one case when we have NULL as ouput
other case when we cannot parse the String value of date to Date Object using SimpleDateFromat, we have to set it to NULL which is existing code.